students_proj/src/main/java/com/musala/simple/students/Main.java

```diff
@@ -0,0 +1,172 @@
+package com.musala.simple.students;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+import com.google.gson.Gson;
+
+public class Main {
+
+	public static void main(String[] args) {
+
+		String path;
+		
+		// Check if any arguments are provided
+		if (args.length == 0) {
+			throw new IllegalArgumentException("Path is missing.");
+		}
+		
+		path = args[0];
```

Comment:
There is no need to declare `String path;` on line 16, you can directly do `String path = args[0];` here.